docs/sphinx/features.rst

```diff
@@ -0,0 +1,41 @@
+.. _features:
+
+=====================
+Features and Examples
```

Comment:
@artv3 I think your examples should be separate from the 'features' page since they really use a combination of features. I think that they should go into the `tutorial.rst` file.

Reply:
Hi @davidbeckingsale , I just moved it over.